pkg/maps/ctmap/ctmap.go

```diff
@@ -48,9 +48,8 @@ type CtKey interface {
 	// Returns human readable string representation
 	String() string
 
-	// Convert converts fields between host byte order and map byte order
-	// if necessary.
-	Convert() CtKey
+	// ToNetwork converts fields to network byte order.
+	ToNetwork() CtKey
```

Comment:
Don't we need to convert it back to host by order? (For example, reading from maps and printing in go)